src/Nethermind/Nethermind.Cli/Modules/DebugCliModule.cs

```diff
@@ -98,9 +98,15 @@ public string GetBlockRlp(long number)
         {
             return NodeManager.Post<string>("debug_getBlockRlp", number).Result;
         }
+        
+        [CliFunction("debug", "setReceiptMigrationPoint")]
+        public bool SetReceiptMigrationPoint(long number)
```

Comment:
call it migrateReceipts or runReceiptsMigration

Reply:
can we add startBlock, endBlock?

Reply:
we could in theory but its not worth it, historically we want to migrate all receipts before some point and I don't see any other use case for it